### PR TITLE
Javascript capstone PR

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
In this PR, we have 

- [x] Found external API.
- [x] Chosen assets.
- [x] Set up the project with Webpack and Jest-.
- [x] Added header and Footer.
- [x] Displayed list of items on the Homepage.
- [x] Displayed Comments pop up with selected item's details.
- [x] Displayed the number of likes for each item on the Homepage.
- [x] Displayed comments for a given item on the Comments pop-up.
- [x] Created feature: add new like.
- [x] Created feature: add a new comment.
- [x] Add all items counter on the Homepage.
- [x] Add comments counter.
- [x] Add tests for the items counter.
- [x] Add tests for comments counter 

link to video demo : https://drive.google.com/file/d/1UB3g54rDrrMYucDM1ptN7SpQzrCfNAIf/view?usp=sharing

### CODE REVIEW

File: src/modules/popup.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      comment,
+    };
+    const CAPSTONE_API_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/CJBclXhG3xsVEAuyngVA/comments';
+    fetch(CAPSTONE_API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify(commentData),
+    })
```
Comment (BenMukebo) [inline src/modules/popup.js:95]: - [ ] Great job the user can post a comment  👍🏻  however 🤔  I'm getting this error when creating a new comment, Please correct this. 💯 👍🏻  

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @Lordkaito, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉

- Git flow is followed. :heavy_check_mark:
- You have followed  JavaScript best practices. ✔️ 
- PR has a good title and summary. ✔️
- The counters functions  is covered with unit tests. ✔️ 
- Project should use ES6 and Webpack
- corectly use async and await with the fetch method to send and receive data. ✔️ 
- Your Design looks awesome, keep rocking :rocket: 

## STATUS: CHANGES REQUESTED  ♻️ 

- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

image



- [ ] Please update your Readme, it will be nice to add and update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. 👍🏻 


-  Please, consider fixing the broken link to the MIT.md file 🥇 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/popup.js (hunk 1 above)
Comment (rachaouldbabaali) [reply src/modules/popup.js:95]: Thank you @BenMukebo

**Review by Shisui6 — APPROVED** [review]
Hi @rachaouldbabaali and @kifle23, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- All requirements met ✅
- Good design ✅


## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**